aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -400,12 +414,28 @@ module AptosFramework::Stake {
     /// Returns if the current validator is still active (can still vote) in the current epoch.
     /// This includes validators that requested to leave but are still in the pending_inactive queue and will be removed
     /// when the epoch starts.
-    public fun is_current_validator(addr: address): bool acquires ValidatorSet{
+    public fun is_current_validator(addr: address): bool acquires ValidatorSet {
         let validator_set = borrow_global<ValidatorSet>(@CoreResources);
         Option::is_some(&find_validator(&validator_set.active_validators, addr)) ||
         Option::is_some(&find_validator(&validator_set.pending_inactive, addr))
     }
 
+    public(friend) fun update_performance_scores(missed_votes: vector<u64>) acquires ValidatorPerformance {
```

Comment:
nit: rename to update_performance_statistics since it is not calculating a single score for ranking the performance

Reply:
Updated